src/d/actor/d_a_spotbox.cpp

```diff
@@ -3,31 +3,78 @@
 // Translation Unit: d_a_spotbox.cpp
 //
 
-#include "d_a_spotbox.h"
+#include "JSystem/JKernel/JKRHeap.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/d_com_inf_game.h"
+#include "m_Do/m_Do_mtx.h"
 #include "dolphin/types.h"
 
+struct daSpotbox_c : public fopAc_ac_c {
+public:
+    inline int create();
+    inline int draw();
+    inline int execute();
+    inline u32 getType();
+public:
+    Mtx mtx;
+}; /* size = 0x2A0 */
+
+int daSpotbox_c::create() {
+    /* Matching 96% */
+    fopAcM_SetupActor(this, daSpotbox_c);
+    float baseScale = this->getType() != 0 ? 100.0f : 1000.0f;
+    this->mScale.x *= baseScale;
+    this->mScale.y *= baseScale;
+    this->mScale.z *= (baseScale * 1.2f);
+    this->current.pos.y += this->mScale.y * 0.5f;
+    this->mCullMtx = ((daSpotbox_c *)this)->mtx;
+    fopAcM_setCullSizeBox(this, -0.5, -0.5f, -0.5f, 0.5f, 0.5f, 0.5f);
+
+    return cPhs_COMPLEATE_e;
+}
```

Comment:
this is the function I cant fully match due to an annoying 'mr' in the wrong place:

<img width="1464" alt="image" src="https://github.com/zeldaret/tww/assets/1191258/7a18c8f6-2c0e-4b8d-af7c-13af23ae9bc5">